File: java/h00.override/Triangle.java
package shape_space;

public class Triangle extends Shape {
    /******* CONSTRUCTORS *******/
    public Triangle(Double base, Double height)  { super(base, height); }
    public Triangle(Integer base, Integer height)  { super(base, height); }

    /******* METHODS *******/
    @Override
    public Double calculateArea() {
      return (super.width * super.height)/2;
    }
}

File: python/h00.override/Triangle.py
#!/usr/bin/env python
from Shape import Shape  # include Shape.py

# class declaration
class Triangle(Shape):
    ###### INITIALIZER ######
    def __init__(self, *args):
        self.base = args[0] / 1.0
        self.height = args[1] / 1.0

    ###### METHODS ######
    def calculateArea(self):
        return (self.base * self.height) / 2
